ddtrace/_trace/trace_handlers.py

```diff
@@ -580,8 +575,10 @@ def _on_django_after_request_headers_post(
 
 
 def _on_botocore_patched_api_call_started(ctx):
+    from ddtrace._trace.utils_botocore.span_tags import set_botocore_patched_api_call_span_tags
+
     span = ctx.span
-    set_patched_api_call_span_tags(
+    set_botocore_patched_api_call_span_tags(
```

Comment:
```suggestion
_imported_modules = {}

def _on_botocore_patched_api_call_started(ctx):
    if "botocore_span_tags" not in _imported_modules:
        from ddtrace._trace.utils_botocore.span_tags import set_botocore_patched_api_call_span_tags
        _imported_modules["botocore_span_tags"] = set_botocore_patched_api_call_span_tags
    
    span = ctx.span
    set_botocore_patched_api_call_span_tags = _imported_modules["botocore_span_tags"](
```

suggesting this because I'm a bit worried for customers using a burst of API calls. Let me know if it makes sense

Reply:
My understanding is that essentially, python is doing this same thing when you attempt to import a package. It checks `sys.modules` and if the module is present, it uses that. If not, then it will actually do the work to import the module.